### PR TITLE
Chore: Introduce GitHub Actions Commitlint workflow

### PR DESCRIPTION
**Pull request by literat**

### CODE REVIEW

File: .github/workflows/commitlint.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  push:
+    branches-ignore:
+      - "main"
+
+jobs:
+  commitlint:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 5
+    env:
+      CI: "true"
```
Comment (OndraM) [inline .github/workflows/commitlint.yaml:13]: This is set by default :)

https://github.blog/changelog/2020-04-15-github-actions-sets-the-ci-environment-variable-to-true/

Comment (literat) [reply .github/workflows/commitlint.yaml]: Thanks. Removed.

File: .github/workflows/commitlint.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+jobs:
+  commitlint:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 5
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 50
+      - uses: actions/setup-node@v1
```
Comment (adamkudrna) [inline .github/workflows/commitlint.yaml:17]: There is [already v2](https://github.com/actions/setup-node/releases).

File: .github/workflows/commitlint.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        with:
+          node-version: 14
+
+      - name: Fetch main
+        run: git fetch --no-tags --depth=50 origin main
+
+      - run: make install
+      - name: Commitlint
```
Comment (adamkudrna) [inline .github/workflows/commitlint.yaml:25]: (Maybe a blank line here…?)

File: .github/workflows/commitlint.yaml [hunk 4]
```diff
@@ -0,0 +1,26 @@
+---
+name: Commitlint
+on:
+  push:
+    branches-ignore:
+      - "main"
```
Comment (adamkudrna) [inline .github/workflows/commitlint.yaml:6]: Quotes are not necessary in this case.

File: .github/workflows/commitlint.yaml [hunk 5]
```diff
@@ -0,0 +1,26 @@
+---
```
Comment (adamkudrna) [inline .github/workflows/commitlint.yaml:1]: Is there a reason for having an empty YML header…?

**Review by adamkudrna — APPROVED** [review]
Cool!

**Comment by adamkudrna** [comment]
@literat It took me two minutes to realize we had already solved this somewhere else and you didn't reply to my today's comments [one month ago](https://github.com/lmc-eu/spirit-design-system/pull/2) 🙈.

File: .github/workflows/commitlint.yaml (hunk 2 above)
Comment (literat) [reply .github/workflows/commitlint.yaml]: Great, updated.

File: .github/workflows/commitlint.yaml (hunk 3 above)
Comment (literat) [reply .github/workflows/commitlint.yaml:26]: Added blank line and `name` for install section.

File: .github/workflows/commitlint.yaml (hunk 4 above)
Comment (literat) [reply .github/workflows/commitlint.yaml]: Ok, updated.

File: .github/workflows/commitlint.yaml (hunk 5 above)
Comment (literat) [reply .github/workflows/commitlint.yaml]: No reason, removed.

**Pull request merged**